### ISSUE TITLE
Python client fails when no config set

### ISSUE THREAD

**Issue by trungly** [launchdarkly/python-client#33]
I'm got the following when I tried the Quickstart tutorial:
```

Traceback (most recent call last):
  File "test.py", line 4, in <module>
    client = LDClient("api-094349dd-49bf-4aa5-91a7-7ad9bbbd1817")
  File "/Users/tly/.virtualenvs/launchdarkly/lib/python2.7/site-packages/ldclient/client.py", line 153, in __init__
    self._store = config.feature_store_class()
AttributeError: 'NoneType' object has no attribute 'feature_store_class'
```

I'll put together a pull request to fix this.

**Comment by jkodumal** [comment]
@trungly thanks for finding this-- released v0.19.2 with your fix in place.

**Comment by trungly** [comment]
:+1:
